corehq/apps/reports/views.py

```diff
@@ -884,45 +884,24 @@ def view_scheduled_report(request, domain, scheduled_report_id):
 @login_and_domain_required
 @require_GET
 def case_details(request, domain, case_id):
-    timezone = get_timezone_for_user(request.couch_user, domain)
-
     try:
         case = get_document_or_404(CommCareCase, domain, case_id)
     except Http404:
         messages.info(request, "Sorry, we couldn't find that case. If you think this is a mistake please report an issue.")
         return HttpResponseRedirect(CaseListReport.get_url(domain=domain))
 
-    try:
-        owner_name = CommCareUser.get_by_user_id(case.owner_id, domain).raw_username
-    except Exception:
-        try:
-            owning_group = Group.get(case.owner_id)
-            owner_name = owning_group.display_name if owning_group.domain == domain else ''
-        except Exception:
-            owner_name = None
-
-    try:
-        username = CommCareUser.get_by_user_id(case.user_id, domain).raw_username
-    except Exception:
-        username = None
-
     return render(request, "reports/reportdata/case_details.html", {
         "domain": domain,
         "case_id": case_id,
         "case": case,
-        "username": username,
-        "owner_name": owner_name,
-        "slug": CaseListReport.slug,
         "report": dict(
             name=case_inline_display(case),
             slug=CaseListReport.slug,
             is_async=False,
         ),
-        "layout_flush_content": True,
-        "timezone": timezone,
         "case_display_options": {
             "display": request.project.get_case_display(case),
-            "timezone": timezone,
+            "timezone": get_timezone_for_user(request.couch_user, domain),
```

Comment:
I tested it out with a bunch of different cases locally (including real ones I'd previously imported from HQ), and it all looks good, so I'll self-merge.  I'll make sure Nate knows that I touched this just in case stuff comes in that seems related.